docs/UsersGuide/source/LAMGrids.rst

```diff
@@ -81,16 +84,19 @@ out with the SRW App, more advanced users may wish to create their own grid for
 a different region and/or with a different resolution.  Creating a user-defined grid requires
 knowledge of how the SRW App workflow functions, in particular, understanding the set of
 scripts that handle the workflow and experiment generation.  It is also important to note that
-user-defined grids are not a supported feature of the current release, however information is
+user-defined grids are not a supported feature of the current release; however information is
 being provided for the benefit of the FV3-LAM community.
 
 With those caveats in mind, this section provides instructions for adding a new grid to the FV3-LAM
-workflow that will be generated using the "ESGgrid" method (i.e. using the regional_esg_grid code
+workflow that will be generated using the "ESGgrid" method (i.e., using the regional_esg_grid code
 in the UFS_UTILS repository, where ESG stands for "Extended Schmidt Gnomonic").  We assume here
 that the grid to be generated covers a domain that (1) does not contain either of the poles and
 (2) does not cross the -180 deg --> +180 deg discontinuity in longitude near the international
 date line.  Instructions for domains that do not have these restrictions will be provided in a future release.
 
+..
+  COMMENT: Do we have these instructions now? Also, would the same process apply to GFDLgrids? 
+
 The steps to add such a grid to the workflow are as follows:
```

Comment:
I don't think those instructions are available yet.  @gsketefian would know about that.  We've not tested creating a GFDLgrid over the poles or across the -/+180 degree longitude line.